src/Components/Facility/CentralNursingStation.tsx

```diff
@@ -48,8 +48,7 @@ export default function CentralNursingStation({ facilityId }: Props) {
       asset_class: "HL7MONITOR",
       ordering: qParams.ordering || "bed__name",
       bed_is_occupied:
-        (qParams.hide_monitors_without_patient ?? "true") === "true" ||
-        undefined,
+        qParams.show_monitors_without_patient === "true" ? false : true,
```

Comment:
If "show monitors without patient" is set to true, `bed_is_occupied` filter should not be set, else it'll show monitors only without monitors and not without + with.

Feel free to change the variable name to `include_monitors_without_patient` to convey the meaning more accuratley.

```suggestion
        qParams.show_monitors_without_patient === "true" ? undefined : true,
```